### ISSUE TITLE
Maybe the original author should be mentioned......

### ISSUE THREAD

**Issue by k3nnyg** [719Ben/Baudrate.py#6]
[http://www.devttys0.com/2012/11/reverse-engineering-serial-ports/](url)

This is where this code originally came from. Did you really put a MIT license file on someone else's code? 

Note this guy references the original source of this code:

[https://github.com/somu1795/baudrate](url)

That's a nice thing to do don't you think? I'm fairly certain that the authors motivation to write and release this was not for you to take credit. It was written and donated to the world to make someones life a little bit easier or to teach and share knowledge. What is your motivation for re-posting it and taking credit for it? Do the right thing or take this down. In fact the author provides a fantastic tutorial on reverse engineering serial ports on embedded devices. He deserves at least credit if not thanks also for the significant time he took to do all this.

NOTE: I am NOT the original author. Right is right and that's why I took the time to write this.

**Comment by biw** [comment]
Hi @k3nnyg, I got this from google code when it was going down, didn't know there was a tutorial. I copied it here since google code was shutting down and I found the code useful at an old job and thought other people might too.

Would love to credit, just couldn't find him when I copied the code to GitHub. The MIT license still lists him which I got from google code at the time, not trying to steal his work at all!

I'll add a link right now, thanks for bringing this to my attention, and for posting the tutorial, I never knew it existed. 😄
